KioskTests/Bid Fulfillment/ConfirmYourBidEnterYourEmailViewControllerTests.swift

```diff
@@ -8,12 +8,23 @@ func testConfirmYourBidEnterYourEmailViewController() -> ConfirmYourBidEnterYour
     return ConfirmYourBidEnterYourEmailViewController.instantiateFromStoryboard(fulfillmentStoryboard).wrapInFulfillmentNav() as! ConfirmYourBidEnterYourEmailViewController
 }
 
+let window = UIWindow()
+
 class ConfirmYourBidEnterYourEmailViewControllerTests: QuickSpec {
     override func spec() {
 
+        beforeSuite {
+            window.makeKeyAndVisible()
+        }
+
         it("looks right by default") {
             let subject = testConfirmYourBidEnterYourEmailViewController()
-            expect(subject).to(haveValidSnapshot())
+            _ = subject.view
+            subject.loadViewProgrammatically()
+            // Highlighting of the text field as it becomes first responder is inconsistent without this line.
+            subject.view.drawViewHierarchyInRect(CGRect.zero, afterScreenUpdates: true)
+
+            expect(subject).to(haveValidSnapshot(usesDrawRect: true))
```

Comment:
Mmmm, no it's required, all the snapshots fail without it because [a key window needs to be set](https://github.com/facebook/ios-snapshot-test-case/blob/fbb2d277cda66350487a88e318cd5a3457738ddd/FBSnapshotTestCase/Categories/UIApplication%2BStrictKeyWindow.m#L18-L23). I'll add a comment.